pom.xml

```diff
@@ -378,6 +378,7 @@
         <module>examples/storm-jms-examples</module>
         <module>examples/storm-rocketmq-examples</module>
         <module>examples/storm-perf</module>
+        <module>storm-checkstyle</module>
```

Comment:
Nit: Move up to the uncategorized modules so this isn't part of the examples list